test/tests/functional/pbs_reservations.py

```diff
@@ -51,6 +51,19 @@ class TestReservations(TestFunctional):
     reservations
     """
 
+    def setUp(self):
+        super().setUp()
+        self._submitted_reservations = []
+        # self.server.manager(MGR_CMD_SET, SERVER, {'log_events': '8191'})
+
+    def tearDown(self):
+        for rid in self._submitted_reservations:
+            try:
+                self.server.delete(rid)
+            except PbsDeleteError:
+                pass
```

Comment:
Won't revert_to_defaults() do this?

Reply:
@bhroam  @RKORavi  The issue was that for tests like `test_resv_server_restart`, which leave an active exclusive reservation on the nodes, the following would happen during the test teardown, delaying the test completion by approximately 3 minutes.
```
2024-02-08 20:24:41,955 INFO     ptl.lib.ptl_object expect on server pdw-tw-suse15-0-s1: state ~ free|provisioning|offline|job-busy node pdw-tw-suse15-0-s1 attempt: 180  got: state = resv-exclusive
```

With the following change I was able to eliminate the `setUp` and `tearDown` methods from `TestReservations`.
```
@@ -1763,7 +1763,7 @@ class PBSTestSuite(unittest.TestCase):
         if 'skip-teardown' in self.conf:
             return
         self.log_enter_teardown()
-        self.server.cleanup_jobs()
+        self.server.cleanup_jobs_and_reservations()
         self.stop_proc_monitor()
 
         for server in self.servers.values():
```
If you find that to be and acceptable solution, I will commit and push the patch.